### PR TITLE
CUID update

### PR DESCRIPTION
**Pull request by IDCalTech**

### CODE REVIEW

File: deployment/GP05MOAS-PG575_Deploy.csv [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+RB1605,,RB1605,,GP05MOAS-PG575-01-CTDGVM000,1,1,2016-06-27T18:58:00,2016-07-05T14:52:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-CTDGVM-09295,50.1487,-144.5076,,200,200,
+RB1605,,RB1605,,GP05MOAS-PG575-02-DOSTAM000,1,1,2016-06-27T18:58:00,2016-07-05T14:52:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-DOSTAM-00499,50.1487,-144.5076,,200,200,
+RB1605,,RB1605,,GP05MOAS-PG575-03-FLORTM000,1,1,2016-06-27T18:58:00,2016-07-05T14:52:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTM-04089,50.1487,-144.5076,,200,200,
+RB1605,,RB1605,,GP05MOAS-PG575-04-FLORTO000,1,1,2016-06-27T18:58:00,2016-07-05T14:52:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTO-01347,50.1487,-144.5076,,200,200,
+RB1605,,RB1605,,GP05MOAS-PG575-05-NUTNRM000,1,1,2016-06-27T18:58:00,2016-07-05T14:52:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-NUTNRM-00711,50.1487,-144.5076,,200,200,
+RB1605,,RB1605,,GP05MOAS-PG575-06-PARADM000,1,1,2016-06-27T18:58:00,2016-07-05T14:52:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-PARADM-50202,50.1487,-144.5076,,200,200,
```
Comment (snwhite) [inline deployment/GP05MOAS-PG575_Deploy.csv:8]: Looks good.

File: deployment/GS05MOAS-PG565_Deploy.csv [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+NBP1511,,NBP1610,,GS05MOAS-PG565-01-CTDGVM000,1,1,2015-12-20T17:46:00,2016-11-21T19:43:00,CGVGS-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-CTDGVM-09287,-54.12519,-89.76248,,200,200,
+NBP1511,,NBP1610,,GS05MOAS-PG565-02-DOSTAM000,1,1,2015-12-20T17:46:00,2016-11-21T19:43:00,CGVGS-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-DOSTAM-00437,-54.12519,-89.76248,,200,200,
+NBP1511,,NBP1610,,GS05MOAS-PG565-03-FLORTM000,1,1,2015-12-20T17:46:00,2016-11-21T19:43:00,CGVGS-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-FLORTM-03815,-54.12519,-89.76248,,200,200,
+NBP1511,,NBP1610,,GS05MOAS-PG565-04-FLORTO000,1,1,2015-12-20T17:46:00,2016-11-21T19:43:00,CGVGS-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-FLORTO-01332,-54.12519,-89.76248,,200,200,
+NBP1511,,NBP1610,,GS05MOAS-PG565-05-NUTNRM000,1,1,2015-12-20T17:46:00,2016-11-21T19:43:00,CGVGS-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-NUTNRM-00661,-54.12519,-89.76248,,200,200,
+NBP1511,,NBP1610,,GS05MOAS-PG565-06-PARADM000,1,1,2015-12-20T17:46:00,2016-11-21T19:43:00,CGVGS-05MOAS-00000,CGVEH-GLDRGP-00565,CGINS-PARADM-50200,-54.12519,-89.76248,,200,200,
```
Comment (snwhite) [inline deployment/GS05MOAS-PG565_Deploy.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]
The 2 glider CSV updates look good.

File: deployment/GP05MOAS-PG575_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/GP05MOAS-PG575_Deploy.csv:8]: Looks good

File: deployment/GS05MOAS-PG565_Deploy.csv (hunk 2 above)
Comment (cdobs) [reply deployment/GS05MOAS-PG565_Deploy.csv:8]: Looks good

**Review by cdobs — COMMENTED** [review]
Both glider files look good.

**Review by snwhite — APPROVED** [review]
GA01SUMO update looks good.

File: deployment/GA01SUMO_Deploy.csv [hunk 3]
```diff
@@ -81,7 +81,7 @@ NBP1510,,NBP1609,,GA01SUMO-RII11-02-FLORDG032,2,1,2015-11-14T21:06:00,2016-11-08
 NBP1510,,NBP1609,,GA01SUMO-RII11-02-FLORDG033,2,1,2015-11-14T21:06:00,2016-11-08T10:10:00,CGMGA-01SUMO-00002,,CGINS-FLORDG-03382,-42.9246,-42.58668,,130,5188,
 NBP1510,,NBP1609,,GA01SUMO-RII11-02-PCO2WC051,2,1,2015-11-14T21:06:00,2016-11-08T10:10:00,CGMGA-01SUMO-00002,,CGINS-PCO2WC-C0121,-42.9246,-42.58668,,40,5188,
 NBP1510,,NBP1609,,GA01SUMO-RII11-02-PCO2WC052,2,1,2015-11-14T21:06:00,2016-11-08T10:10:00,CGMGA-01SUMO-00002,,CGINS-PCO2WC-C0122,-42.9246,-42.58668,,80,5188,
-NBP1510,,NBP1609,,GA01SUMO-RII11-02-PCO2WC053,2,1,2015-11-14T21:06:00,2016-11-08T10:10:00,CGMGA-01SUMO-00002,,,-42.9246,-42.58668,,,5188,Not Deployed
+#NBP1510,,NBP1609,,GA01SUMO-RII11-02-PCO2WC053,2,1,2015-11-14T21:06:00,2016-11-08T10:10:00,CGMGA-01SUMO-00002,,,-42.9246,-42.58668,,,5188,Not Deployed
```
Comment (cdobs) [inline deployment/GA01SUMO_Deploy.csv:84]: Looks good

**Review by cdobs — APPROVED** [review]
All looks good

**Pull request merged**